public/apps/configuration/panels/internal-user-edit/internal-user-edit.tsx

```diff
@@ -147,6 +148,18 @@ export function InternalUserEdit(props: InternalUserEditDeps) {
     }
   };
 
+  const descriptionData = [
+    {
+      renderComponent: (
+        <EuiText size="xs" color="subdued">
+          The security plugin includes an internal user database. Use this database in place of, or
+          in addition to, an external <br /> authentication system such as LDAP or Active Directory.{' '}
```

Comment:
Nit: Why is there a line break here?

Reply:
Linter rules prevent a line from getting too long, and if it's too long the linter would automatically add {' '} in order to show space between the end of the text and link. Clean up will be done separate to this change.